net-analyzer/sslsplit/sslsplit-0.5.4.ebuild

```diff
@@ -22,23 +22,29 @@ else
 fi
 
 RDEPEND="
-	elibc_musl? ( sys-libs/fts-standalone )
 	dev-libs/libevent[ssl,threads]
-	dev-libs/openssl:0="
+	dev-libs/openssl:0=
+	net-libs/libnet:1.1
+	elibc_musl? ( sys-libs/fts-standalone )"
 DEPEND="${RDEPEND}
 	test? ( dev-libs/check )"
+BDEPEND=""
```

Comment:
These empty variables can be dropped from the ebuild. Not gonna ask you to fix it now anymore, but bear it in mind next time. LGTM.

Reply:
I thought it's mandatory. I'll remember it, thanks a lot!